### PR TITLE
feat(divider): create divider component

### PR DESCRIPTION
**Pull request by pepeladeira**
A thin horizontal line that separates content areas built on top of `<hr>`.

Also apply the `Hr` component in `VercelInviteUser` preview

### CODE REVIEW

File: packages/divider/src/divider.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export const Divider = React.forwardRef<DividerElement, Readonly<DividerProps>>(
+  ({ style, ...props }, forwardedRef) => (
+    <hr
+      ref={forwardedRef}
+      style={{
+        width: "100%",
+        border: "none",
+        borderTop: "1px solid #eaeaea",
+        margin: "26px 0",
```
Comment (zenorocha) [inline packages/divider/src/divider.tsx:16]: This margin shouldn't be on a component level, it should be on the template level only.

File: packages/divider/src/divider.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        ...style,
+      }}
+      {...props}
+    ></hr>
```
Comment (zenorocha) [inline packages/divider/src/divider.tsx:20]: This can be a self-closing tag instead

File: packages/divider/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@react-email/divider",
+  "version": "0.0.2",
```
Comment (zenorocha) [inline packages/divider/package.json:3]: 0.0.1

**Review by zenorocha — CHANGES_REQUESTED** [review]
What do you think of renaming to `<Hr />` instead? That way, we can keep it as familiar to regular HTML as possible.

File: packages/divider/src/divider.tsx (hunk 1 above)
Comment (pepeladeira) [reply packages/divider/src/divider.tsx]: Okay!

File: packages/divider/src/divider.tsx (hunk 2 above)
Comment (pepeladeira) [reply packages/divider/src/divider.tsx]: great

**Review by zenorocha — APPROVED** [review]

**Comment by zenorocha** [comment]
This is awesome, thanks @pepeladeira!

**Pull request merged**